### PR TITLE
Blog on ignition data support

### PR DESCRIPTION
**Pull request by karmab**
pending on ignition PR https://github.com/kubevirt/kubevirt/pull/1657/files

### CODE REVIEW

File: _posts/2018-11-20-ignition-support.markdown [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+virtualmachine "myvm1" created
+```
+
+### Step 4
+
+Enjoy
+
+## Summary
+
+Ignition Support brings the ability to run coreos/rhcos distros on Kubevirt and to customize them at boot time
```
Comment (rwsu) [inline _posts/2018-11-20-ignition-support.markdown]: It wasn't clear to me what ignition support was about until I read this. Perhaps this should be stated in the beginning.

File: _posts/2018-11-20-ignition-support.markdown [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+## Introduction
+
+Ignition is a new provisioning utility designed specifically for CoreOS/RhCoreos. At the the most basic level, it is a tool for manipulating disks during early boot. This includes:
+
+- partitioning disks
+- formatting partitions
+- writing files (regular files, systemd units, networkd units)
+- configuring users and their associated ssh public keys
+
+Recently we added support for ignition data in Kubevirt so that ignition data can be embedded in a vm specification
```
Comment (rwsu) [inline _posts/2018-11-20-ignition-support.markdown]: missing "."

File: _posts/2018-11-20-ignition-support.markdown [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Ignition support is still needed in the guest operating system
```
Comment (rwsu) [inline _posts/2018-11-20-ignition-support.markdown]: missing "."

File: _posts/2018-11-20-ignition-support.markdown [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+## Example
+
+We assume that you already have a Kubernetes or Openshift cluster running with KubeVirt installed.
+
+### Step 1
+
+Create The following VM spec in the file *myvm1.yml*
```
Comment (rwsu) [inline _posts/2018-11-20-ignition-support.markdown]: missing ":"?

File: _posts/2018-11-20-ignition-support.markdown [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+virtualmachine "myvm1" created
+```
+
+### Step 4
+
+Enjoy
+
+## Summary
+
+Ignition Support brings the ability to run coreos/rhcos distros on Kubevirt and to customize them at boot time
```
Comment (rwsu) [inline _posts/2018-11-20-ignition-support.markdown]: Missing "."

File: _posts/2018-11-20-ignition-support.markdown [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+Create the VM:
+
+```
+$ kubectl apply -f myvm1.yml
+virtualmachine "myvm1" created
+```
+
+### Step 4
+
+Enjoy
```
Comment (rwsu) [inline _posts/2018-11-20-ignition-support.markdown]: Missing "."

File: _posts/2018-11-20-ignition-support.markdown (hunk 2 above)
Comment (karmab) [reply _posts/2018-11-20-ignition-support.markdown]: done

File: _posts/2018-11-20-ignition-support.markdown (hunk 3 above)
Comment (karmab) [reply _posts/2018-11-20-ignition-support.markdown]: done

File: _posts/2018-11-20-ignition-support.markdown (hunk 4 above)
Comment (karmab) [reply _posts/2018-11-20-ignition-support.markdown]: done

**Pull request merged**